bindings/java/src/blocking_operator.rs

```diff
@@ -159,3 +160,27 @@ fn intern_delete(env: &mut JNIEnv, op: &mut BlockingOperator, path: JString) ->
     let path = env.get_string(&path)?;
     Ok(op.delete(path.to_str()?)?)
 }
+
+/// # Safety
+///
+/// This function should not be called before the Operator are ready.
+#[no_mangle]
+pub unsafe extern "system" fn Java_org_apache_opendal_BlockingOperator_info<'local>(
```

Comment:
Please avoid adding features while refactoring unless you have a strong reason on doing so.

Reply:
I'll create a new pr to add this.